server/rest/src/main/java/org/infinispan/rest/server/operations/mediatypes/printers/UTF8OutputPrinter.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.rest.server.operations.mediatypes.printers;
+
+import java.nio.charset.Charset;
+
+import org.infinispan.CacheSet;
+import org.infinispan.rest.server.operations.mediatypes.OutputPrinter;
+
+public class UTF8OutputPrinter implements OutputPrinter {
+
+   @Override
+   public byte[] print(String cacheName, CacheSet<?> keys) {
+      return keys.toString().getBytes(Charset.forName("UTF-8"));
```

Comment:
Same thing I don't know if we can rely on the toString implementation of every `CacheSet` impl to be the way we want.